packages/fether-react/src/stores/createAccountStore.js

```diff
@@ -17,32 +17,33 @@ import getParityWordlist from './utils/getParityWordlist';
 
 const debug = Debug('createAccountStore');
 
-export const SIGNER_ACCOUNTS_LS_KEY = `${LS_PREFIX}::paritySignerAccounts`;
 const DERIVATION_PATH = "m/44'/60'/0'/0/0";
-const MIN_PARITY_SIGNER_RECOVERY_WORDS = 11;
-const MAX_PARITY_SIGNER_RECOVERY_WORDS = 24;
 const ETHEREUM_ADDRESS_LENGTH = 40;
 const JSON_VERSION = 3;
+const MIN_PARITY_SIGNER_RECOVERY_WORDS = 11;
+const MAX_PARITY_SIGNER_RECOVERY_WORDS = 24;
+export const SIGNER_ACCOUNTS_LS_KEY = `${LS_PREFIX}::paritySignerAccounts`;
 
 export class CreateAccountStore {
   @observable
-  isImport = false; // Are we creating a new account, or importing via phrase?
+  address = null;
 
   @observable
-  jsonString = null;
+  bip39Phrase = null; // 12 to 24-words seed phrase
```

Comment:
nitpick: since we're modifying the lines, let's correct the grammar while we're at it and write `12 to 24-word seed phrase` :sweat_smile: